figures/views.py

```diff
@@ -106,6 +109,20 @@ class CommonAuthMixin(object):
     )
 
 
+class StaffUserOnDefaultSiteAuthMixin(object):
+    '''Provides a common authorization base for the Figures API views
+    TODO: Consider moving this to figures.permissions
+    '''
+    authentication_classes = (
+        BasicAuthentication,
```

Comment:
Do we really need the HTTP Basic authentication? Usually we should be good with just the session and the token auth.

```suggestion
        BasicAuthentication,
```

Reply:
Not necessary. I'll remove it when I get to this PR after I'm all prepped for the conference

Reply:
Actually, it may come in handy. I'm going to hold of on removing it. Card created to not lose track of this: https://github.com/appsembler/figures/issues/86